src/sentry/newsletter/base.py

```diff
@@ -24,3 +25,6 @@ def update_subscription(self, user, **kwargs):
     def create_or_update_subscription(self, user, **kwargs):
         kwargs['create'] = True
         return self.update_subscription(user, **kwargs)
+
+    def optout_email(self, email, property):
+        return None
```

Comment:
I think it might be better for this to raise a `NotImplementedError`, since if it's not implemented by a backend but is called we should get notified about it.

Reply:
fair enough, i'll update